core/src/transaction_builder.rs

```diff
@@ -27,9 +27,9 @@ pub type CreateAddressOutputs = (Address, TaprootSpendInfo);
 
 #[derive(Debug, Clone)]
 pub struct TransactionBuilder {
-    verifiers_xonly_pks: Vec<XOnlyPublicKey>,
-    verifiers_pks: Vec<PublicKey>,
-    network: Network,
+    _verifiers_xonly_pks: Vec<XOnlyPublicKey>,
```

Comment:
There are some places this was used before, that are now commented out. If they currently are not needed, we can remove this struct completely.

Reply:
Yeah, I think @ekrembal will also agree that we do not need that struct anymore. But I am not exactly sure what to do with it, or how to use the functions from **TransactionBuilder**.

Reply:
Yes, we don't need TransactionBuilder as a struct. instead of using TransactionBuilder:: we should use transaction_builder::

Reply:
But i would say let's merge withdrawal-test first, and then get rid of TransactionBuilder struct, then merge to main

Reply:
Removed in [fa2ea4c](https://github.com/chainwayxyz/clementine/pull/240/commits/fa2ea4cdbb1fb4ed4b667452af19f92eeb90527d).